src/components/ReportActionItem/ReportPreview.js

```diff
@@ -89,7 +99,7 @@ const defaultProps = {
 const ReportPreview = (props) => {
     const reportAmount = CurrencyUtils.convertToDisplayString(ReportUtils.getMoneyRequestTotal(props.iouReport), props.iouReport.currency);
     const managerEmail = props.iouReport.managerEmail || '';
-    const managerName = ReportUtils.getDisplayNameForParticipant(managerEmail, true);
+    const managerName = ReportUtils.isPolicyExpenseChat(props.chatReport) ? ReportUtils.getPolicyName(props.chatReport) : ReportUtils.getDisplayNameForParticipant(managerEmail, true);
```

Comment:
We're pulling the chatReport object to simply get the managerName? Is it not possible to get this from the IOUReport?

Reply:
Ah, right. What about the reportAction? IOUActions should contain participants under [action.originalMessage.participants ](https://github.com/Expensify/App/blob/762554742330769dc29dbb476acd7564d62439d3/src/components/ReportActionItem/IOUPreview.js#L144-L146)